node/src/components/storage/tests.rs

```diff
@@ -1674,7 +1674,14 @@ fn check_force_resync_with_marker_file() {
     // Add a couple of blocks into storage.
     let first_block = Block::random(&mut harness.rng);
     put_complete_block(&mut harness, &mut storage, Box::new(first_block.clone()));
-    let second_block = Block::random(&mut harness.rng);
+    let second_block = loop {
```

Comment:
nit: we can use `Block::random_with_specifics()`  to avoid doing a loop here.

Reply:
I discussed this a bit with @georgepisaltu and we had a common understanding that the `random_with...` Block API is a bit too clunky. In this particular case, I would need to provide more specific parameters than just height.

My plan for [sync leaper testing](https://github.com/casper-network/casper-node/issues/3445) is to prepare a builder:
```
TestBlockBuilder::random().with_height().with_something_else().build()
```
so if you're not strong about this change I'll merge as is.